app/views/find/courses/_advice.html.erb

```diff
@@ -7,9 +7,8 @@
     <%= govuk_link_to "chat online", t("get_into_teaching.url_online_chat") %>
     <%= t("get_into_teaching.opening_times") %> (except public holidays).</p>
 
-  <p class="govuk-body">Find out about the different
-  <%= govuk_link_to("ways to train", t("get_into_teaching.url_ways_to_train")) %>
-  if you do not have a degree, or are already an experienced teacher.</p>
+  <p class="govuk-body">Find out about the different ways to train <%= govuk_link_to("if you do not have a degree", t("get_into_teaching.url_ways_to_train_no_degree")) %>
+  , or <%= govuk_link_to("are already an experienced teacher", t("get_into_teaching.url_ways_to_train_experienced_teacher")) %>.</p>
```

Comment:
There's an extra space here before the comma:
<img width="654" alt="image" src="https://github.com/DFE-Digital/publish-teacher-training/assets/47917431/deeaf625-7b48-4370-b63d-f320f856e4c9">

Reply:
I've pushed a new commit for this